src/Microsoft.VisualStudio.Editor.Razor/VisualStudioWorkspaceAccessor.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    internal abstract class VisualStudioWorkspaceAccessor
```

Comment:
Yeah, this is exactly what I had in mind to paper over the differences between VS and VS4Mac